neural-redis.dockerfile

```diff
@@ -0,0 +1,47 @@
+#----------------------------------------------------------------------------------------------
+# This work is subject to the terms of the Redis Source Available License
+# https://redislabs.com/wp-content/uploads/2019/09/redis-source-available-license.pdf
```

Comment:
This is actually incorrect, Neural Redis is under the BSD license: https://github.com/antirez/neural-redis/blob/master/COPYING